VERSION.txt

```diff
@@ -1,4 +1,5 @@
 jetty-10.0.16-SNAPSHOT
+ + Jan is now using Intellij! 
```

Comment:
Typo: Intellij -> IntelliJ

+1 to remember this.

Reply:
Fixed. I have updated the PR title so this will be part of the auto generated release notes!